pages/common/vagrant.md

```diff
@@ -0,0 +1,19 @@
+# vagrant
+
+> Configure lightweight, reproducible, and portable development environments
+
+- Create Vagrantfile in current folder with the given Vagrant box
+
+`vagrant init hashicorp/precise32`
```

Comment:
Good point, we've had this discussion before about specific examples vs. generic tokens. In this case I believe it's clearer to use the token syntax, but maybe put the example in the command description to make the format evident (i.e. xxxx/yyyy).

Alternatively, we could use `{{xxxx}}/{{yyyy}}`, replacing xxxx and yyyy by the proper names of those components.